test/integration/publishing/publish-organisations.test.js

```diff
@@ -109,4 +109,11 @@ describe('publish organisations', () => {
     await publish()
     expect(mockSendMessage).toHaveBeenCalledTimes(1)
   })
+
+  test('should re-publish if previously published dataset updated', async () => {
+    await publish()
+    await db.organisation.update({ updated: new Date(2022, 8, 5, 15, 30, 10, 121) }, { where: { sbi: 123456789 } })
+    await publish()
+    expect(mockSendMessage).toHaveBeenCalledTimes(2)
+  })
 })
```

Comment:
consider changing description of previous test to something like `should not publish same organisation on second run if record has not been updated after previous run`